### PR TITLE
fix tests locally

### PR DESCRIPTION
**Pull request by ramenhog**
## Issue Number
#60 

## Purpose/Implementation Notes
* Fix tests for `helpers.js` and `App.js` locally

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

* [x] Bugfix (non-breaking change which fixes an issue)

## Functional tests
* Tests run locally

### CODE REVIEW

File: src/components/Modal/ModalManager.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import Button from '../Button';
 
 import './Modal.scss';
 
-Modal.setAppElement('#root');
+if (process.env.NODE_ENV !== 'test') Modal.setAppElement('#root');
```
Comment (dongbohu) [inline src/components/Modal/ModalManager.js]: Does CircleCI set `process.env.NODE_ENV` to "test"?

Comment (ramenhog) [reply src/components/Modal/ModalManager.js]: When the command `yarn test` runs, it sets it to "test" apparently

Comment (dongbohu) [reply src/components/Modal/ModalManager.js]: So the test will fail with `Modal.setAppElement('#root')`?

Comment (ramenhog) [reply src/components/Modal/ModalManager.js]: Yes: https://github.com/reactjs/react-modal/issues/632

**Pull request merged**